src/projectOutline.ts

```diff
@@ -546,7 +561,15 @@ export class ProjectOutline implements vscode.TreeDataProvider<BaseNode> {
         }
 
         const updates: BaseNode[] = [];
-        existing.updateCodeModel(model, { ...ctx, nodesToUpdate: updates, folder });
+        existing.updateCodeModel(
```

Comment:
Am I reading this right and this still has 3 parameters even though the method signature only has two parameters now?

Reply:
It is called WorkspaceFolderNode::updateCodeModel, which has three parameters now.